dapps/templates/demo/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "embark-dapp-template-demo",
+  "version": "4.0.0-beta.0",
+  "description": "Demo DApp for embark",
+  "files": [
+    "app",
+    "config",
+    "contracts",
+    "dot.gitignore",
+    "dot.npmrc",
+    "embark.json",
+    "test"
+  ],
+  "scripts": {
+    "ci": "npm run qa",
+    "clean": "npm run reset",
+    "package": "npm pack",
+    "qa": "npm-run-all test package",
+    "reset": "npx embark-reset && npx rimraf embark-*.tgz package",
+    "test": "npx embark test"
```

Comment:
why npx instead of the one in deps?

Reply:
`npx` will use the one in `node_modules` if it's there. The yarn workspace sets up symlinks (linux, macos) and shim scripts (windows) automatically during `yarn install` at the root, when done manually or as part of `yarn reboot[:full]`.

Reply:
TIL